src/ios/lib/manager.ts

```diff
@@ -78,11 +78,20 @@ export class ClientManager {
   }
 
   async getDebugserverClient() {
-    return this.getServiceClient(
-      'com.apple.debugserver',
-      DebugserverClient,
-      true,
-    );
+    try {
+      // iOS 14 added support for a secure debug service so try to connect to that first
+      return this.getServiceClient(
```

Comment:
Must await for errors to be thrown into this `catch` statement (otherwise they are bubbled up to the caller).

```suggestion
      return await this.getServiceClient(
```